.reuse/dep5

```diff
@@ -1,11 +1,11 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: <YOUR-REPO-NAME>
-Upstream-Contact: <YOUR-CONTACT (MAIL ADDRESS ETC.)>
-Source: <https://github.com/sap/YOUR-REPO-NAME>
-Disclaimer: The code in this project may include calls to APIs ("API Calls") of
+Upstream-Name: kyma-environment-broker
```

Comment:
this probably should be a `cloud-manager` instead

Reply:
Changed.